docs/articles/nunit/release-notes/framework.md

```diff
@@ -4,6 +4,73 @@ uid: frameworkreleasenotes
 
 # Framework Release
 
+## NUnit 4.0.0-alpha.1 - TBA
```

Comment:
Just adding a note that we should ensure `TBA` is replaced with a date before merging.